src/Model/User.php

```diff
@@ -96,6 +96,36 @@ class User
 
 	private static $owner;
 
+	private static function welcome_text(): string
```

Comment:
Allow providing the nickname and password as arguments:

```suggestion
	private static function welcome_text(string $nickname, string $password): string
```